Dockerfile

```diff
@@ -1,14 +1,12 @@
-ARG ALPINE=alpine:3.11
-FROM ${ALPINE} AS alpine
+FROM ghcr.io/oracle/oraclelinux:7-slim AS builder
 ARG ARCH
 ARG KUBERNETES_RELEASE=v1.17.0
 WORKDIR /bin
 RUN set -x \
- && apk --no-cache add curl \
  && curl -fsSLO https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_RELEASE}/bin/linux/${ARCH}/kubectl \
  && chmod +x kubectl
 
-FROM scratch
-COPY --from=alpine /bin/kubectl /bin/kubectl
+FROM ghcr.io/oracle/oraclelinux:7-slim
```

Comment:
We should leave the final image based on `scratch`, there's no need to use an OS base here and using scratch means we never have to deal with CVEs in the container.

Reply:
Okay, sounds good